multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangacatalog/MangaCatalogGenerator.kt

```diff
@@ -36,6 +36,7 @@ class MangaCatalogGenerator : ThemeSourceGenerator {
         SingleLang("Read Tokyo Ghoul Re & Tokyo Ghoul Manga Online", "https://ww8.tokyoghoulre.com", "en", className = "ReadTokyoGhoulReTokyoGhoulMangaOnline", overrideVersionCode = 1),
         SingleLang("Read Tower of God Manhwa Manga Online", "https://ww1.readtowerofgod.com", "en", className = "ReadTowerOfGodManhwaMangaOnline", overrideVersionCode = 2),
         SingleLang("Read Vinland Saga Manga Online", "https://ww1.readvinlandsaga.com", "en"),
+        SingleLang("Read Black Clover Manga Online", "https://ww7.readblackclover.com/", "en", className = "ReadBlackCloverMangaOnline", pkgName = "readblackclovermangaonline"),
```

Comment:
Also don't put this at the end it should be placed according to alphabetical order
```suggestion
        SingleLang("Read Black Clover Manga Online", "https://ww7.readblackclover.com", "en"),
```

Reply:
You didn't remove them class and pkg name